emercoin-ubuntu/azuredeploy.json

```diff
@@ -195,7 +188,7 @@
           "fileUris": [
             "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/emercoin-ubuntu/configure-emercoin.sh"
           ],
-          "commandToExecute": "[concat('sh configure-emercoin.sh ', parameters('adminUsername'))]"
+          "commandToExecute": "[concat('sh configure-emercoin.sh ', '\"', parameters('adminUsername'), '\" \"',  parameters('adminPassword'), '\"')]"
```

Comment:
If you're passing in the password to the script, the recommendation to pass it in is to using protected settings https://github.com/Azure/azure-quickstart-templates/blob/875d139c16c9c023dce519e6dd48c707e3473346/splunk-on-ubuntu/splunk-2disk-resources-loop.json#L172